q2_fmt/tests/test_engraftment.py

```diff
@@ -1121,10 +1122,76 @@ def test_rename_features_with_blank_label(self):
                                        reference_column="Ref",
                                        subject_column="subject")
         _rename_features(data=feature_peds_df, level_delimiter=";")
-        print(feature_peds_df)
         Fs1 = feature_peds_df.set_index("id").at['Feature 1 __',
                                                  'subject']
         Fs2 = feature_peds_df.set_index("id").at['Feature 2',
                                                  'subject']
         self.assertEqual("1", Fs1)
         self.assertEqual("2", Fs2)
+
+
+class TestBoot(TestBase):
```

Comment:
Are there relevant boundary conditions that should be tested, such as a single donor, a single recipient, PEDS=1, PEDS=0, ...)?